Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocCheckBox.css

```diff
@@ -3,38 +3,64 @@
 .bocCheckBox
 {
   margin-right: 0.5em;
+  white-space: nowrap;
+}
+
+.bocCheckBox input
+{
+  position: absolute;
+  z-index: -1;
+}
+
+.bocCheckBox img
+{
+  margin: 2px;
   vertical-align: middle;
+  height: 16px;
+  width: 16px;
+}
+
+.bocCheckBox input + span::before
+{
+  margin: 2px;
+  visibility: visible;
+  background-image: url('../Image/sprite.svg#CheckBoxFalse');
+  width: 16px;
+  height: 16px;
   display: inline-block;
-  white-space: nowrap;
+  content: "";
+  z-index: 1;
+  position: relative;
 }
 
-.bocCheckBox.readOnly
+.bocCheckBox input[checked] + span::before
 {
+  background-image: url('../Image/sprite.svg#CheckBoxTrue');
 }
 
-.bocCheckBox.disabled
+.bocCheckBox,
+.bocCheckBox > input + span
 {
+  display: flex;
+  align-items: center;
 }
 
-.bocCheckBox input
+/* Fixes for element focus */
+
+.bocCheckBox span[role=checkbox]:focus
 {
-  margin-left: 4px;
-  margin-right: 3px;
-  padding: 0px;
-  vertical-align: -0.1em;
+  outline: none;
 }
 
-.bocCheckBox span + span
+.blink .bocCheckBox span[role=checkbox]:focus > img,
+.blink .bocCheckBox input:focus + span::before
 {
-  vertical-align: middle;
+  box-shadow: 0 0 0 1px black;
+  border-radius: 1px;
 }
 
-.bocCheckBox img
+.mozilla .bocCheckBox span[role=checkbox]:focus > img,
+.mozilla .bocCheckBox input:focus + span::before
 {
-  margin-left: 0px;
-  margin-right: 0;
-  padding: 0px;
-  vertical-align: middle;
-  height: 16px;
-  width: 16px;
+  outline: 1px dotted black;
 }
```

Comment:
I think we agreed that we do the outline with `box-shadow`. But I don't know if that should be done in this PR or if it will be done later.🤔

Reply:
A new and unified version comes later. 

For now, we should stick with the browser-default, either by not overriding it, or, if needed, done in the most straight-forward way. Mozilla does a blue outline/border + boxshadow in pastel blue, Chrome a black border/outline.

Reply:
> Mozilla does a blue outline/border + boxshadow in pastel blue

Reference please. I'm seeing a black dotted border.

Reply:
It depends on the type of the selected element. The (default) outline of an input is blue, while the (default) outline of a span is dotted.
![image](https://user-images.githubusercontent.com/48289242/129543055-46d0491f-1da7-4d20-9704-1ebad02726de.png)
![image](https://user-images.githubusercontent.com/48289242/129543629-938b18e0-3d88-464f-907a-43773deb168d.png)